go.mod

```diff
@@ -1,15 +1,54 @@
 module github.com/coinbase/mongobetween
 
-go 1.13
+go 1.18
```

Comment:
Upgrading to 1.18 since it's the minimum required version for the latest Mongo driver; doing a `go mod tidy` afterwards introduced the second `require` block.